nixos/doc/manual/release-notes/rl-2405.section.md

```diff
@@ -95,6 +95,8 @@ The pre-existing [services.ankisyncd](#opt-services.ankisyncd.enable) has been m
 
 - [GNS3](https://www.gns3.com/), a network software emulator. Available as [services.gns3-server](#opt-services.gns3-server.enable).
 
+- [Gatus](https://github.com/TwiN/gatus), an automated developer-oriented status page. Available as [services.gatus](#opt-services.gatus.enable).
```

Comment:
Isn't this supposed to `nixos/doc/manual/release-notes/rl-2411.section.md` ?

Reply:
Sorry for the delay, I have moved the release notes to 24.11